### ISSUE TITLE
Small fixes to the wiki

### ISSUE THREAD

**Issue by pietromarchesi** [pharmbio/sciluigi#46]
I went through the 'Using' page of the wiki and found a few things that may need to be fixed. 

- Currently, the example replaces the `foo` string in `foo.txt` with `bar`, but then saves to `foo.txt.bar.txt`. I think it would make sense to save either to `foo.txt` or to `bar.txt`.
- The `ex()` method uses `in` which is actually a keyword, and thus results in a syntax error. It also refers to `self.out_bar()` which has been renamed to `self.out_replace()` in the rest of the example. 

I can fix this and submit a PR if needed?

**Comment by samuell** [comment]
> I can fix this and submit a PR if needed?

Absolutely, many thank in advance! :)

**Comment by pietromarchesi** [comment]
Done! I also figured you need an extra slash in the `sed` command. Should work now!

**Comment by pietromarchesi** [comment]
I was also wondering why you set 
```python
# Here we have one input, a "foo file":
in_foo = None
```
In `MyFooReplacer`, it seems to me that it's not necessary to set `in_foo` to `None`.

**Comment by samuell** [comment]
> In MyFooReplacer, it seems to me that it's not necessary to set in_foo to None (at least, I see the logic of it, but the workflow ran fine without it).

Yeah, this is a practice we adopted mainly for documentary purposes, to make it clearer which are the in-ports for the task, and so that it would show up when using auto-completion in the editor, to aid when designing the workflows.

**Comment by samuell** [comment]
> Done!

Many thanks!
